src/presentation/tenant-iam-presenter-factory.ts

```diff
@@ -25,6 +25,7 @@ export class TenantIamPresenterFactory {
     format: OutputFormat,
     includeAncestors: boolean,
     meshTenants: MeshTenant[],
+    userFilterString?: string,
```

Comment:
t: you can tell that this is a sketchy design decision because it has to be "wired" through so many function calls to get into the right position. Usually, better decisions are those that reduce the argument count.
Sadly I don't know how to do better. :D So I would says its okay for now but at some point imo we need to refactor this section anyways when we e.g. unify tenant view/display.

Reply:
Yeah I think the meshTenants[] array is already an example of this too: I think the `collie user list` command at the end of the day deserves its own presenter layer. Right now it's built on top of the existing `collie tenant iam` command presenter layer.